src/app/page.js

```diff
@@ -1,7 +1,59 @@
+'use client'
+import CollegeResult from './components/CollegeResult'
+import jsonData from '../../script/web-scraping-script-main/university_data'
+import { useState, useEffect } from 'react'
 export default function Home() {
+  const [open, setOpen] = useState(false)
+  const [collegesData, setCollegesData] = useState([])
+  const [searchData, setSearchData] = useState([])
+  async function handler() {
+    setCollegesData(jsonData)
+  }
+  useEffect(() => {
+    handler()
+  }, [])
```

Comment:
Similar to a comment mentioned earlier the `handler()` and `useEffect()` can be removed and we can directly load the file.